specification/paths.json

```
@@ -242,6 +242,9 @@
   "/import-service-rates": {
     "$ref": "./paths/ImportServiceRates.json"
   },
+  "/import-shipment": {
+    "$ref": "./paths/ImportShipment.json"
+  },
```

I propose to rename this so it's a bit more defined / less generic.
```suggestion
  "/import-registered-shipment": {
    "$ref": "./paths/ImportRegisteredShipment.json"
  },
```